### ISSUE TITLE
Model training

### ISSUE THREAD

**Issue by abdur4373** [XPFly1989/FCRN#4]
Hello

I see from your code that you are using pre trained model "NYU_ResNet-UpProj.npy". After training model for 50 epochs it start giving better result. But when we start training why initial depth predictions are not good enough according to pre-trained weights from above file. I want to fine tune the FCRN model on my own dataset by initializing with the pre-trained model. Any help on this please.

**Comment by XPFly1989** [comment]
Hi @abdur4373 
The pre-trained model is well fitted to the data that the model used to train, if you have different data (e.g. outdoor scene), the predictions may not good, in my understanding.
So just initializing with the pre-trained weights and use your own data to train. I think you can get a better result.

**Comment by abdur4373** [comment]
Thanks @XPFly1989  for your reply. Actually I have prepared my own data set of indoor scene in my environment and want to train model on that. I am freezing all other layers except for the up projection blocks and the result is not so good. Even I trained it on as small data set as 600 images and achieved 82 percent accuracy but the results were not good visually.
I donot know the reason of that maybe you can suggest me something.
And the images I want to train are approximately 6k. 
The pretrained weights with NYU are even performing better.
    batch_size = 32
    learning_rate = 1.0e-3
    monentum = 0.9
    weight_decay = 0.0005
    num_epochs = 70
 optimizer = torch.optim.SGD(filter(lambda p: p.requires_grad, model.parameters()), lr=learning_rate,
                                momentum=monentum, weight_decay=weight_decay)
and lr is halved after 10 epochs.


![MS_LAB_269_unfilled](https://user-images.githubusercontent.com/46248215/60284606-5fb44480-9925-11e9-9032-a6e4f90f9062.png)

Validation depth image
![Screenshot from 2019-06-27 21-37-28](https://user-images.githubusercontent.com/46248215/60284080-19121a80-9924-11e9-9c73-2689e3ed9831.png)

rgb image
![Screenshot from 2019-06-27 21-44-59](https://user-images.githubusercontent.com/46248215/60284489-1e239980-9925-11e9-9481-e6e043afafb7.png)

**Comment by LiaoChangyue** [comment]
@abdur4373 hello，I want to experiment on a single picture, can you provide the code to run on a single picture. thank you very much
